.github/workflows/install-test-react-native.yml

```diff
@@ -6,63 +6,133 @@ on:
+      - name: Configure ccache
+        if: ${{ runner.os == 'macOS' }}
+        run: ccache --set-config="compiler_check=content"
+
+      # TODO: Remove if this becomes unneeded in the future
       - name: Invoke the simulator (making subsequent "open -a Simulator" calls work)
+        if: ${{ matrix.platform == 'ios' }}
         run: open -a ${{ env.DEVELOPER_DIR }}/Contents/Developer/Applications/Simulator.app/Contents/MacOS/Simulator
+
+      - name: Install dependencies of the CLI
+        run: npm ci
+
+      - name: Initialize app
+        # Using "--skip-bundle-install" to let the setup-ruby action install the bundle
+        # Using "--skip-pod-install" to ensure it happens after setup-ruby has executed
+        run: npm run init -- --skip-bundle-install --skip-pod-install --realm-version ${{ matrix.realm-version }} --react-native-version ${{ matrix.react-native-version }} --engine ${{ matrix.engine }}
+
+      - name: CocoaPod cache
+        uses: actions/cache@v3
+        with:
+          path: '**/Pods'
+          key: cocoa-pods-${{ matrix.platform }}-r@${{ matrix.realm-version }}-rn@${{ matrix.react-native-version }}-${{ matrix.new-architecture && 'new' || 'old' }}-arch
+
+      - uses: ruby/setup-ruby@v1
+        if: ${{ matrix.platform == 'ios' }}
+        with:
+          bundler-cache: true
+          working-directory: install-tests/react-native/app
+
+      - if: ${{ matrix.platform == 'ios' }}
+        run: pod install
+        working-directory: install-tests/react-native/app/ios
+
+      - uses: actions/setup-java@v3
+        if: ${{ matrix.platform == 'android' }}
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
       - name: Run test (iOS)
-        run: ./run-ios.sh
-      # - name: Run test (Android)
-      #   uses: reactivecircus/android-emulator-runner@v2
-      #   with:
-      #     api-level: 29
-      #     target: google_apis
-      #     script: ./run-android.sh
+        if: ${{ matrix.platform == 'ios' }}
+        run: npm test -- --platform ios
+
+      - name: Run test (Android)
+        if: ${{ matrix.platform == 'android' }}
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 29
+          target: google_apis
+          script: npm test -- --platform android
+          working-directory: install-tests/react-native
+
+  announce:
+    needs: install
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Post result to Slack
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          # For posting a rich message using Block Kit
+          payload: |
+            {
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "Install test *${{ needs.install.result }}*\n${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

Comment:
Is this enough information?  Would be great to see a table of the matrix and what passed or failed.

Reply:
I think you're right .. I did try, but I can try a bit harder :)